mobile/lib/services/backup.service.dart

```diff
@@ -331,31 +349,35 @@ class BackupService {
                 livePhotoFile.lengthSync(),
                 filename: livePhotoTitle,
               );
-              req.files.add(livePhotoRawUploadData);
-              fileSize += livePhotoFile.lengthSync();
+              final livePhotoReq = MultipartRequest(
+                req.method,
+                req.url,
+                onProgress: req.onProgress,
+              )
+                ..headers.addAll(req.headers)
+                ..fields.addAll(req.fields);
+
+              livePhotoReq.files.add(livePhotoRawUploadData);
+
+              // Send live photo only if the non-motion part is successful
+              if (response.statusCode == 200 || response.statusCode == 201) {
+                final data = await response.stream.bytesToString();
+                final mediaResponse = jsonDecode(data);
+                final assetId = mediaResponse['id'].toString();
+                livePhotoReq.fields['livePhotoVideoId'] = assetId;
+
+                response = await httpClient.send(
```

Comment:
I'm not totally sure, as I'm not too familiar with dart code - but I think this is performing the upload of the livePhoto part of the asset. If so, make sure the form-field is correct (i.e. if you choose to drop it and use assetData instead) 

Additionally, you'll need to add 1 more request after this one, to perform the linking of the live-photo to the still-photo. That can be performed using the standard PUT /api/asset/:id call, and just specify the a single property, which is 'livePhotoVideoId'. I had this change in one of my PRs, but I think it got lost in the shuffle. To complete this, you'll need to modify the UpdateAssetBase DTO to allow PUT request to specify the livePhotoVideoId for a given asset. 

Since there's already so many changes - might as well consider making the names consistent.... (is|has|apply|play|stop)motionPhoto is used in a bunch of .dart code, some web code, and some microservice code. In most other places its called 'livePhotoVideo'. 

It appears that Android calls this "Motion Photos" and iOS calls it "Live Photos" which is where the confusion comes from. But LivePhotoVideo is not a thing ;-)